### PR TITLE
Mule 11856 fix

### PR DESCRIPTION
**Pull request by fernandezlautaro**
MULE-11856: @Xml annotation attributes renaming, fix broken build

### CODE REVIEW

File: src/main/java/org/mule/runtime/api/component/DefaultComponentIdentifier.java [hunk 1]
```diff
@@ -53,19 +53,19 @@ static ComponentIdentifier parseComponentIdentifier(String componentIdentifier)
...
+    return new DefaultComponentIdentifier.Builder().withPrefix(namespace).withName(identifier).build();
   }
 
   public static class Builder implements ComponentIdentifier.Builder {
 
     private DefaultComponentIdentifier componentIdentifier = new DefaultComponentIdentifier();
 
     /**
-     * @param namespace namespace identifier of the mule language extensions module
+     * @param prefix namespace identifier of the mule language extensions module
```
Comment (pablokraan) [inline src/main/java/org/mule/runtime/api/component/DefaultComponentIdentifier.java]: mule > Mule

File: src/main/java/org/mule/runtime/api/meta/model/XmlDslModel.java [hunk 2]
```diff
@@ -114,15 +114,15 @@ public String getSchemaVersion() {
...
    * @return The extension's namespace
    */
-  public String getNamespace() {
-    return namespace;
+  public String getPrefix() {
+    return prefix;
   }
 
   /**
    * @return The namespace URI
```
Comment (pablokraan) [inline src/main/java/org/mule/runtime/api/meta/model/XmlDslModel.java]: Fix javadoc

File: src/main/java/org/mule/runtime/api/component/DefaultComponentIdentifier.java (hunk 1 above)
Comment (canoasan) [inline src/main/java/org/mule/runtime/api/component/DefaultComponentIdentifier.java]: really?

Comment (pablokraan) [inline src/main/java/org/mule/runtime/api/component/DefaultComponentIdentifier.java]: do you write names starting with lower case or upper case?

**Pull request merged**